app/sprinkles/core/src/Bakery/SetupFontAwesomeCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * UserFrosting (http://www.userfrosting.com)
+ *
+ * @link      https://github.com/userfrosting/UserFrosting
+ * @copyright Copyright (c) 2019 Alexander Weissman
+ * @license   https://github.com/userfrosting/UserFrosting/blob/master/LICENSE.md (MIT License)
+ */
+
+namespace UserFrosting\Sprinkle\Core\Bakery;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use UserFrosting\System\Bakery\BaseCommand;
+
+class SetupFontAwesomeCommand extends BaseCommand
+{
+    /**
+     * @var string Path to the build/ directory
+     */
+    protected $buildPath = \UserFrosting\ROOT_DIR . \UserFrosting\DS . \UserFrosting\BUILD_DIR_NAME;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setName('setup:font-awesome');
+
+        // the short description shown while running "php bakery list"
+        $this->setDescription('Switch between free/pro versions of Font Awesome');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        // Display header,
+        $this->io->title("UserFrosting's Font Awesome Setup Wizard");
+
+        // Display nice explanation and ask which method to use
+        $this->io->write('This setup will configure Font Awesome 5 Pro. Font Awesome 5 Free is already included with UserFrosting.');
+
+        if (!$this->io->confirm('Would you like to continue?', false)) {
+            exit;
+        }
+
+        $this->io->note("The Authentication Key will be saved in `{$this->buildPath}/.npmrc`");
+        $this->io->write('Please enter your Font Awesome Authentication Key to begin. This will be used to download the neccessary packages.');
```

Comment:
This is actually called `Pro npm Package Token` in the Font Awesome account details.

Reply:
I removed the bakery command for simplicity sake. I noted the changes and have it saved in a different branch which I will work on sometime in the future...